modules/claims_api/app/controllers/concerns/claims_api/v2/claims_requests/tracked_items.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'bgs_service/tracked_item_service'
```

Comment:
why the require here? our creation of local_bgs and the require was in the v2/application_controller, as well as the instantiation of the service.

Reply:
> why the require here? our creation of local_bgs and the require was in the v2/application_controller, as well as the instantiation of the new TrackedItemService service.

```"exception": "uninitialized constant ClaimsApi::TrackedItemService"```

Reply:
still seeing this when calling claims#show

Reply:
we either need that require in the app controller, or we put this back - but if we put the require back here, we should initialize the service in this concern, which is warranted due to the functionality of this concern. we don't use it outside of this.